tutorials/introductory/usage.py

```diff
@@ -357,16 +357,16 @@ def my_plotter(ax, data1, data2, param_dict):
 #
 # By default, Matplotlib should automatically select a default backend which
 # allows both interactive work and plotting from scripts, with output to the
-# screen and/or to a file, so at least initially you will not need to worry
+# screen and/or to a file, so at least initially, you will not need to worry
 # about the backend.  The most common exception is if your Python distribution
```

Comment:
The interjection feels like it's "at least initially".

As "fixed", the comma makes the sentence read like "so at least initially" is the interjection, but you can't remove that whole clause from the sentence and it still make sense. 

I would leave it out, personally, but you could also do ", so, at least initially," (eww) or, my prefered:

```suggestion
# screen and/or to a file, so you will not need to worry about the backend, at
# least initially. The most common exception is if your Python distribution
```